testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5361,6 +5361,28 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("array_length(null)");
   }
 
+  /** Tests {@code MAP_KEYS} function from Spark. */
+  @Test void testMapKeysFunc() {
+    final SqlOperatorFixture f = fixture()
+        .setFor(SqlLibraryOperators.MAP_KEYS)
```

Comment:
I think it could be good to write a test that fails if library is not set to Spark (you only want it to work for Spark).

Reply:
added @tanclary